src/ast/query.rs

```diff
@@ -2197,6 +2200,8 @@ pub enum JoinOperator {
         match_condition: Expr,
         constraint: JoinConstraint,
     },
+    /// STRAIGHT_JOIN (non-standard)
```

Comment:
Could we include a link to the mysql docs for reference?

Reply:
Of course, done!